### PR TITLE
(GH-58) Update documentation

### PR DESCRIPTION
**Pull request by glennsarti**
Previously the extension documentation was in the root directory however it
should reside in the client directory.  This commit:
- Moves the extension readme into the client directory
- Adds the puppet logo
- Updates the Server documentation with new command line arguments and logging
  format
- Adds a stub README in the root of the repo with information about the server
  and client.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,89 +8,51 @@ TODO: Need correct marketplace ID
...
+### Run the language server
 
-![Example of features](docs/assets/language_server.gif)
+Follow the instructions in the server documentation - [How to run the Language Server for Develoment](server/READMEmd#How_to_run_the_Language_Server_for_Development). Ensure you use the `--timeout=0` and `--no-stop` arguments so the server does not stop.
 
-## Requirements
+### Run the client
 
-- Puppet Agent
+* Ensure a nodejs is installed
```
Comment (jpogran) [inline README.md]: remove the 'a' before nodejs

File: README.md [hunk 2]
```diff
@@ -8,89 +8,51 @@ TODO: Need correct marketplace ID
...
 
-- [Syntax highlighting](#Syntax_Highlighting)
-- [Code snippets](#Code_snippets)
-- IntelliSense for resources, parameters and more
-- Import from `puppet resource` directly into manifests
-- Node graph preview
+### Run the language server
 
-![Example of features](docs/assets/language_server.gif)
+Follow the instructions in the server documentation - [How to run the Language Server for Develoment](server/READMEmd#How_to_run_the_Language_Server_for_Development). Ensure you use the `--timeout=0` and `--no-stop` arguments so the server does not stop.
```
Comment (jpogran) [inline README.md]: This may need to be changed now that #56 is merged

Comment (glennsarti) [inline README.md]: This is in the `How to run the client and server for development` section so it seems applicable.  Perhaps I need to make this more obivious?

**Comment by glennsarti** [comment]
@jpogran updated.

File: README.md (hunk 2 above)
Comment (jpogran) [inline README.md]: Nope, Im good with that explanation

**Pull request merged**